### ISSUE TITLE
Could not find module `ember-searchable-select/controllers/application`

### ISSUE THREAD

**Issue by jcloutz** [tedconf/ember-searchable-select#20]
After following the installation instructions i receive the following error which stops the application from running with ember-cli 1.13.12. I also test with 1.13.8 and received the same error.

```
Error while processing route: orders.index Could not find module `ember-searchable-select/controllers/application` imported from `tracker-ui/controllers/application` Error: Could not find module `ember-searchable-select/controllers/application` imported from `tracker-ui/controllers/application`
    at requireFrom (http://localhost:4200/assets/vendor.js:126:13)
    at reify (http://localhost:4200/assets/vendor.js:113:22)
    at mod.state (http://localhost:4200/assets/vendor.js:156:17)
    at tryFinally (http://localhost:4200/assets/vendor.js:37:14)
    at requireModule (http://localhost:4200/assets/vendor.js:155:5)
    at Ember.DefaultResolver.extend._extractDefaultExport (http://localhost:4200/assets/vendor.js:70598:20)
    at resolveOther (http://localhost:4200/assets/vendor.js:70330:32)
    at superWrapper (http://localhost:4200/assets/vendor.js:33728:20)
    at exports.default._emberRuntimeSystemObject.default.extend.resolveController (http://localhost:4200/assets/vendor.js:15743:19)
    at exports.default._emberRuntimeSystemObject.default.extend.resolve (http://localhost:4200/assets/vendor.js:15588:43)
```

Any help would be much appreciated.

**Comment by brenna** [comment]
@jcloutz Thanks for flagging - this bug only appeared when `pods` structure was not in use.  

Patched in [v0.3.1](https://github.com/tedconf/ember-searchable-select/releases/tag/v0.3.1). Should be all good if you bump your version in `package.json` to `"ember-searchable-select": "^0.3.1"` and `npm install`.

**Comment by jcloutz** [comment]
Awesome, I'll give it a try.
